airbyte-server/src/main/java/io/airbyte/server/errors/NotFoundExceptionMapper.java

```diff
@@ -38,9 +37,12 @@
 
   @Override
   public Response toResponse(NotFoundException e) {
-    LOGGER.error("Not found exception", e);
+    // Would like to send the id along but we don't have access to the http request anymore to fetch it
+    // from. Come back to this.
```

Comment:
can we write an issue for this and convert this into a TODO with the issue tag so we don't forget?